fastlane/Fastfile

```diff
@@ -629,13 +628,13 @@ REPOSITORY_NAME="WordPress-Android"
     version=options[:version]
     name="wpandroid-#{version["name"]}.aab"
     apk_name=universal_apk_name(version)
-    aab_file="org.wordpress.android-#{options[:flavor]}-#{options[:buildType]}.aab"
+    aab_file="org.wordpress.android-#{options[:app]}-#{options[:flavor]}-#{options[:buildType]}.aab".downcase
```

Comment:
The filename generated is all lowercase and the casing needs to match to pick up the file on CI.